test/Interop/Cxx/static/inline-static-member-var-irgen.swift

```diff
@@ -0,0 +1,13 @@
+// Test that static member variables are handled properly.
+
+// RUN: %empty-directory(%t)
+// RUN: %target-swift-frontend -I %S/Inputs -emit-ir -enable-cxx-interop -Xcc -std=c++17 %s -o %t/static-member-var.ir
+// RUN: %FileCheck < %t/static-member-var.ir %s
+
+// CHECK: @_ZN22WithInlineStaticMember12staticMemberE = linkonce_odr global i32 256, comdat, align 4
```

Comment:
Do you want to also test the code generation pattern for the read and write into the variable?

Reply:
Done.